models/dataset_test.go

```diff
@@ -25,8 +26,8 @@ func TestCreateDataset(t *testing.T) {
 
 			b, err := json.Marshal(inputDataset)
 			if err != nil {
-				log.ErrorC("Failed to marshal test data into bytes", err, nil)
-				os.Exit(1)
+				t.Logf("failed to marshal test data into bytes, error: %v", err)
```

Comment:
I think it would be more concise to use `So(err, ShouldBeNil)`

Reply:
It's not a test assertion, the json marshaller has literally failed when it shouldn't